client/my-sites/site-settings/jetpack-credentials/credentials-setup-flow/setup-footer.jsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'gridicons';
+import Popover from 'components/popover';
+
+class SetupFooter extends Component {
+	componentWillMount() {
+		this.setState( { showPopover: false } );
+	}
+
+	togglePopover = () => this.setState( { showPopover: ! this.state.showPopover } );
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+
+			<CompactCard className="credentials-setup-flow__footer">
+				<a
+					onClick={ this.togglePopover }
+					ref="popoverLink"
+					className="credentials-setup-flow__footer-popover-link"
+				>
+					<Gridicon icon="help" size={ 18 } className="credentials-setup-flow__footer-popover-icon" />
+					{ translate( 'Why do I need this?' ) }
+				</a>
+				<Popover
+					context={ this.refs && this.refs.popoverLink }
+					isVisible={ get( this.state, 'showPopover', false ) }
```

Comment:
Not sure if we need `get()` here as if `this.state.showPopover` doesn't exist it will just be `undefined` and thus _falsey_. My guess is that it would evaluate properly for any expected values. `this.state` should always exist in these kinds of components (vs. functional stateless components)